www/src/components/navigation/themeToggleButton.astro

```diff
@@ -0,0 +1,81 @@
+<div
+  class="flex bg-t3-purple-200/50 dark:bg-slate-700 p-2 w-fit mx-auto rounded-lg gap-3"
+>
+  <label class="cursor-pointer text-t3-purple-500 dark:text-slate-900">
```

Comment:
i made both icons the same color, can change the sun back to gold if you don't like it

Reply:
I like this more tbh

Reply:
yea me too, keeping it then :)